File: Java/Area_Equilateral.java
package com.lev;

import java.util.Scanner;

public class Area_Equilateral {
    public static void main(String[] args) {
        System.out.print("Enter side :");
        Scanner in = new Scanner(System.in);
        float a = in.nextFloat();
        float area =(float) Math.sqrt(3)*a*a/4;
        System.out.println("Area : "+area);
    }
}

File: C++/Area_Equilateral.cpp
//Area of equilateral triangle
#include<iostream>
#include<cmath>
using namespace std;
int main(){
    int s;
    cout<<"Enter side :";
    cin>>s;
    float area = sqrt(3)*s*s/4;
    cout<<"Area : "<<area;
    return 0;
}
